### DISCUSSION TITLE
i have waiting for 6 hrs. How can i fix it?

### DISCUSSION THREAD

**Discussion by haroon889** [Chia-Network/chia-blockchain#8390] [Support]
i have waiting for 6 hrs. How can i fix it?
![Screenshot 2021-08-22 124743](https://user-images.githubusercontent.com/84036790/132496666-e79fade0-4038-4bdc-8048-72153d11c8a0.png)

**Comment by loppefaaret** [comment]
could you try and first, check your `debug.log` in your users folder `~\.chia\mainnet\log\` and see if there is any `ERROR` or `WARNING` listed ?
and if you could try closing down Chia, and browse to `C:\Users\<USERNAME>\AppData\Roaming\Chia Blockchain\Local Storage\` and delete the `leveldb` folder - the `AppData` folder is normally hidden, but can be accessed by typing in `%appdata%` in the address bar and hitting ENTER
then open up Chia again, and see if it starts showing the plots under the "Plots" tab ?

**Comment by bigd3435** [comment]
ok i am having the same issue and tried the above and still the same

**Reply by loppefaaret** [reply]
thank you for reporting back - do you see any reports in the `debug.log` file mentioned above ?

**Reply by bigd3435** [reply]
if you look a little further down the page you will see i have posted about this issue and also included my debug logs. there is no error that i can see

**Comment by haroon889** [comment]
You mean this right?

There is any  problem? i really have no idea

ในวันที่ พฤ. 9 ก.ย. 2021 เวลา 05:22 loppefaaret ***@***.***>
เขียนว่า:

**Comment by haroon889** [comment]
เมื่อ พฤ. 9 ก.ย. 2564 เวลา 10:49 bigd3435 ***@***.***>
เขียนว่า:

**Comment by BrandtH22** [comment]
@haroon889 , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

Note the newer versions of the chia client have newer plotters that are much quicker dependent on your devices hardware.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
